crates/bitwarden-core/src/auth/jwt_token.rs

```diff
@@ -31,7 +31,7 @@ pub enum JWTTokenParseError {
 }
 
 impl FromStr for JWTToken {
```

Comment:
If we're renaming `JwtTokenParseError` we should probably also rename `JWTToken` to be `JwtToken`